getting-started.md

```diff
@@ -5,156 +5,93 @@ nav_title: Getting Started
 app that uses [Material Design](https://www.google.com/design/spec/material-design/introduction.html).
 
-The code for your app is in `my_app/lib/main.dart`.
-To learn more about how to build apps with Flutter, please see the
-[tutorial](/tutorial/).
-
-## Setting up your Android device
-
-(Don't have an Android device? You can skip down to
-[Setting up your Mac for iOS development](#setting-up-your-mac-for-ios-development).)
-
-Currently Flutter requires an Android device running
-Jelly Bean, v16, 4.1.x or later.
-
- - Install [Android Studio](https://developer.android.com/sdk/) and run through
-   enough of the first-run experience to install the Android SDK. (You shouldn't
-   need to install a JDK.)
-
- - Enable developer mode on your device by visiting `Settings > About phone` and
-   tapping the `Build number` field seven times.
+In the project directory, the code for your app is in `my_app/lib/main.dart`.
 
- - Enable `Android debugging` in `Settings > Developer options`.
+## 2. Run your Flutter app
 
- - Using a USB cable, plug your phone into your computer. If prompted on your
-   device, authorize your computer to access your device.
+Use the `flutter run` command to run your Flutter app on all connected
+devices and simulators. (The `flutter devices` command will list connected devices and
+simulators.)
 
- - Check that `adb` lists your device with `adb devices -l`.
+To run your app from the command-line:
 
-By default, Flutter uses the version of the Android SDK where your `adb` tool is based. If
-you want Flutter to use a different installation of the Android SDK, you must set the
-`ANDROID_HOME` environment variable to that specific installation directory.
+1. Open a terminal and change directories to the root of your app (the same directory that
+contains the `pubspec.yaml` file for your project).
+2. Run the following command.
 
-## Running a Flutter application on Android
+  ```
+  $ flutter run
+  ```
 
-You can use the `flutter run` command to run your Flutter app on all connected
-devices and simulators.  `flutter devices` will list connected devices and
-simulators.
+Alternatively, if you are using the [Atom editor](https://atom.io/) with the
+[Flutter package](https://atom.io/packages/flutter), you can start your Flutter app by selecting
+the `/lib/main.dart` file in the project and pressing the Run App shortcut in the main editor
+screen. The shortcut depends on the operating system of the development machine you are using:
 
-First, change directories to the root of your app (the same directory that
-contains the `pubspec.yaml` file for your project).
+* **Linux**: `Ctrl+R`
+* **Mac**: `Command+R`
 
-To run your app, use:
 
-```
-$ flutter run
-```
+If everything works, you should see your starter app on your Android device or iOS Simulator.
 
-If everything works, you should see your starter app
-on your Android device.
+<table>
```

Comment:
this isn't a table, it's just two images side by side. Can we make the <figure> here inline-block?